contracts/lib/LibMath.sol

```diff
@@ -12,4 +12,21 @@ library LibMath {
     function abs(int256 x) internal pure returns (int256) {
         return x > 0 ? int256(x) : int256(-1 * x);
     }
+
```

Comment:
I am fairly certain the PRB math library has quite efficient implementation of these. Also since fundingRate will most likely end up being a WAD value (since the price data its based off is WAD values) we will want to eventually use that

Reply:
actually re thinking, the fact that they are WAD values shouldn't matter. Its more a question of which implementation is more efficient

Reply:
@raymogg Honestly all for a non-homemade solution for these -- I'd prefer to not have to write them myself. I agree about the correctness not being impacted and that we should then prioritise efficiency.